requirements.txt

```diff
@@ -25,3 +25,4 @@ tf_slim
 tqdm
 moviepy
 Pillow>=7.1
+openvino-dev==2022.1.0
```

Comment:
Is this necessary in the requirements? This is the file that is parsed when GitHub runs CI, maybe we could leave it out. Please tell me if I'm missing something :)

Reply:
Sure, it can be removed